<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>500</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>屏幕使用时间「代码见三」</string>
  </property>
  <property name="windowIcon">
   <iconset resource="ScreenTime.qrc">
    <normaloff>:/img/img/32.ico</normaloff>:/img/img/32.ico</iconset>
  </property>
  <property name="iconSize">
   <size>
    <width>32</width>
    <height>32</height>
   </size>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QListWidget" name="listWidget"/>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>400</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>帮助</string>
    </property>
    <addaction name="actionOpenAppDir"/>
    <addaction name="actionViewHistory"/>
   </widget>
   <addaction name="menu"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionOpenAppDir">
   <property name="text">
    <string>打开安装目录</string>
   </property>
  </action>
  <action name="actionExitApp">
   <property name="text">
    <string>退出程序</string>
   </property>
  </action>
  <action name="actionSelfStarting">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>开机自启动</string>
   </property>
  </action>
  <action name="actionViewHistory">
   <property name="text">
    <string>查看历史</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="ScreenTime.qrc"/>
 </resources>
 <connections/>
</ui>
